Desktop/widgets/listmodelfactorsform.h

```diff
@@ -43,14 +44,15 @@ class ListModelFactorsForm : public ListModel
 	Terms					termsEx(const QString& what)									override;
 	void					initFactors(const std::vector<std::tuple<std::string, std::string, std::vector<std::string> > > &factors);
```

Comment:
I think a typedef for factors vector is in order:
```
typedef FactorVec std::vector<std::tuple<std::string, std::string, std::vector<std::string> > >;
```
Maybe also one for the elements?
Because this is pretty much unreadable.
Maybe a struct is even better, not sure about that though.